sql-plugin/src/main/scala/org/apache/spark/sql/rapids/execution/python/GpuArrowEvalPythonExec.scala

```diff
@@ -555,7 +555,7 @@ case class GpuArrowEvalPythonExec(
 
     // cache in a local to avoid serializing the plan
     val inputSchema = child.output.toStructType
-    val pythonOutputSchema = StructType.fromAttributes(resultAttrs)
+    val pythonOutputSchema = StructType.fromAttributes(udfs.map(_.resultAttribute))
```

Comment:
nit: Great catch on this by the way. Could we add some comments here explaining what is happening and why we are not using `resultAttrs` that was passed in? If I made the mistake before then a comment will hopefully help someone not break it going back again, especially if it only shows up for databricks.

Reply:
Good suggestion, updated.